src/server/rest/v1/validator/ChargingStationValidator.ts

```diff
@@ -89,6 +91,12 @@ export default class ChargingStationValidator extends SchemaValidator {
     return data;
   }
 
+  public validateChargingStationActionStopTransactionReq(data: HttpChargingStationCommandRequest): HttpChargingStationCommandRequest {
```

Comment:
I haven't noticed but I would expect a typed response here like HttpChargingStationStartTransactionRequest with the args prop typed.

Reply:
done